game/src/debug/debug_plugin.rs

```diff
@@ -0,0 +1,15 @@
+use bevy::prelude::*;
+
+pub struct DebugPlugin;
+
+//add the system(s) you wish to use to see debug info
+impl Plugin for DebugPlugin {
+    fn build(&self, app: &mut App) {
+        app.add_systems(Update, print_test);
+    }
+}
+
+//Function to test if debugger is added correctly
```

Comment:
Like above, use doc strings here to bind your comment to the function.

Reply:
updated